packages/block-library/src/pullquote/block.json

```diff
@@ -26,6 +26,13 @@
 	"supports": {
 		"anchor": true,
 		"align": [ "left", "right", "wide", "full" ],
+		"background": {
+			"backgroundImage": true,
+			"backgroundSize": true,
+			"__experimentalDefaultControls": {
+				"backgroundImage": true
```

Comment:
Should we collapse the background image tools panel by default? 

I'm tending towards "yes" - there are quite a few controls here.

Reply:
Ignore this. It's pretty compact

Reply:
Yeah it doesn't bother me! Also helps with discoverability.